randovania/games/planets_zebeth/logic_database/Brinstar.json

```diff
@@ -305,6 +305,15 @@
                                             "amount": 1,
                                             "negate": false
                                         }
+                                    },
+                                    {
+                                        "type": "resource",
+                                        "data": {
+                                            "type": "tricks",
+                                            "name": "IceAbuse",
+                                            "amount": 1,
+                                            "negate": false
+                                        }
```

Comment:
btw this pickup is unreachable trickless if you don't specify `or hi jump boots`. Could you change this to `Ice Beam and (IceAbuse or Hi Jump Boots)`?

Reply:
> btw this pickup is unreachable trickless if you don't specify or hi jump boots

this isnt an issue per se. You dont *have* to have a game where every item is reachable tricklesss.

Reply:
tho it's better if it's reachable